lib/prometheus/client/label_set_validator.rb

```diff
@@ -7,6 +7,7 @@ module Client
     class LabelSetValidator
       # TODO: we might allow setting :instance in the future
       BASE_RESERVED_LABELS = [:job, :instance, :pid].freeze
+      LABEL_NAME_REGEX = /\A[a-zA-Z_][a-zA-Z0-9_]*\Z/
```

Comment:
The prom docs you linked to say `:` is also allowed...
Are we explicitly not allowing it for some reason? Or was that an omission?

Reply:
Ignore me, that's for metrics names, not label names.